advisories/github-reviewed/2024/09/GHSA-78vg-7v27-hj67/GHSA-78vg-7v27-hj67.json

```diff
@@ -7,12 +7,8 @@
     "CVE-2024-45592"
   ],
   "summary": "auditor-bundle vulnerable to Cross-site Scripting because name of entity does not get escaped",
-  "details": "### Summary\nUnescaped entity property enables Javascript injection.\n\n### Details\nI think this is possible because %source_label% in twig macro is not escaped. Therefore script tags can be inserted and are executed.\n\n### PoC\n- clone example project https://github.com/DamienHarper/auditor-bundle-demo\n- create author with FullName <script>alert()</script>\n- delete author\n- view audit of authors\n- alert is displayed\n\n### Impact\npersistent XSS. JS can be injected and executed.\n",
+  "details": "### Summary\nUnescaped entity property enables Javascript injection.\n\n### Details\nI think this is possible because %source_label% in twig macro is not escaped. Therefore script tags can be inserted and are executed.\n\n### PoC\n- clone example project https://github.com/DamienHarper/auditor-bundle-demo\n- create author with FullName <script>alert()</script>\n- delete author\n- view audit of authors\n- alert is displayed\n\n### Impact\npersistent XSS. JS can be injected and executed.",
   "severity": [
-    {
-      "type": "CVSS_V3",
-      "score": "CVSS:3.1/AV:N/AC:L/PR:L/UI:R/S:C/C:H/I:L/A:L"
-    },
```

Comment:
@Gonzalo1987 why this removal?

Reply:
It was a mistake, the form by default put the CVSS_V4 and change the options, I've just resubmit with CVSS_V3, I'm neither familiar with the CVSS, sorry!!!